README.md

```diff
@@ -1,2 +1,91 @@
 # berbix_flutter_demo
-Demo app for integrating the berbix flutter SDK
+
+Demonstrates how to use the berbix_flutter plugin.
+# Building
+
+## Android
+
+`flutter build appbundle`
+
+## iOS
+
+`flutter build ios --release --no-codesign`
+
+# Add Dependency
+To begin we'll need to add `berbix_flutter` as a dependency to our project. You can accomplish this by executing:
+
+```Bash
+flutter pub add berbix_flutter
+```
+
+from your project directory. That should update your **pubspec.yml** like so:
+
+```yaml
+# Other Settings
+# ...
+dependencies:
+  flutter:
+    sdk: flutter
+  berbix_flutter: ^1.0.0
+```
+
+After we've updated our configuration, we can fetch the new packages by running:
+
+```Bash
+flutter pub get
+```
+
+# Integrating
```

Comment:
Looks like a lot of this README is pulled over from the one in `berbix/berbix_flutter`. The contents of this README should mostly explain how to get the demo app up and running, so I think a lot of this can be pulled out